mkdocs/index.md

```diff
@@ -116,9 +130,15 @@ template: home.html
             data_apps.value = res.filter(item=>item.type !== 'infra')
 
             data_apps.value.forEach(item=>{
-              item.tags.forEach(tag => tagsSet.add(tag));
+              if(item.support_type !==''){
```

Comment:
I think this condition is not needed. The valid values of "support_type" field are "Enterprise" and "Community". If it's not specified, it's automatically set to "Community" so we don't need to add another check here.